src/main/java/mezz/jei/network/PacketHandlerClient.java

```diff
@@ -35,7 +35,7 @@ public void onPacket(NetworkEvent.ServerCustomPayloadEvent event) {
 			if (player != null) {
 				packetHandler.readPacketData(packetBuffer, player);
 			}
-		} catch (Exception e) {
+		} catch (Throwable e) {
```

Comment:
I think catch `RuntimeException` is actually good here, we don't want to catch memory errors and other serious exceptions that could make debugging a crash very confusing.